test/createExamples.js

```diff
@@ -24,6 +24,8 @@ const favicons = require('../dist/index.js'),
         orientation: "any",                       // Default orientation: "any", "natural", "portrait" or "landscape". `string`
         scope: "/",                               // set of URLs that the browser considers within your app
         start_url: "/?homescreen=1",              // Start URL when launching the application from a device. `string`
+        preferRelatedApplications: false,         // Should the browser prompt the user to install the native companion app. `boolean`
```

Comment:
Do we write it always in HTML file? Maybe we can add test?

Anyway thanks for the PR! Good job!